package/MDAnalysis/core/AtomGroup.py

```diff
@@ -1116,15 +1230,15 @@ def bonds(self):
         only bonds which are entirely contained by the AtomGroup use
         u.bonds.atomgroup_intersection(ag, strict=True)
 
-        .. versionadded 0.9.0
+        .. versionadded:: 0.9.0
+        .. versionchanged:: 0.9.2
```

Comment:
Replace all the versionchanged:: 0.9.2 with 0.10.0 (I think – this might be a sufficiently big change to go to 0.10.0)